environments/host_vars/website.yml

```diff
@@ -0,0 +1,7 @@
+---
+ansible_ssh_host: 165.227.4.15
+ansible_ssh_port: 22
+ansible_ssh_user: root
+email: pythondev.slack@gmail.com
+systemd_enabled: true
+travis_ci: false
```

Comment:
Not really sure why I made this comment. Maybe I was a bit confused.
it looks ok to me to this here.